realm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,3 +4,4 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
+set JAVA_OPTS="-Xmx512m"
```

Comment:
I just got another Java Heap space error. I'll try to add the above to `gradle.properties`